packages/visual-dev/src/components/Tag/Styles.ts

```diff
@@ -0,0 +1,33 @@
+import { css } from "styled-components";
+
+export const base = css`
+  display: flex;
+  flex-direction: row;
+  background: #f9f9f9;
+  box-sizing: border-box;
+  align-items: center;
+  width: max-content;
+  border: 1px solid #e2e2e2;
+  border-radius: 2px;
+`;
+
+export const textSegment = css`
+  box-sizing: border-box;
+  border-left: 1px solid #e2e2e2;
+  padding: 3px 7px;
+  font-weight: 400;
+  font-size: 14px;
+  font-style: normal;
+  line-height: 20px;
+  font-family: "Helvetica Neue", Helvetica, sans-serif;
+  color: #232323;
+`;
+
+export const iconSegment = css`
```

Comment:
One small update, change the cursor to a pointer when hovering over the button section of the tag to indicate a click action.

Reply:
Haha we talked about this during pair-programming. Technically the pointer cursor is supposed to only be used for hyperlinks (basically to indicate you're leaving the page) and correct hover states should be what indicates a click action/interaction, but this rule is broken a lot in other sites so I guess we should decide where to draw the line. I'm kind of a purist with cursors but that's just me.

Reply:
I do think there should be a hover state for the icon segment though (darker background or w/e)

Reply:
Good call, I made changes for hover state :+1: